### ISSUE TITLE
drastic no menu in new update 20240205

### ISSUE THREAD

**Issue by vvzpaco** [rg35xx-cfw/rg35xx-cfw.github.io#102]
the button to open the menu still doesn't work in drastic in the new update, even with the _es_systems_nds.cfg_ file, it doesn't work ;(

**Comment by docloulou** [comment]
Same on my side, drastic mapping is not working at all. games are working but cant play because no mapping. (even with the es_systems_nds.cfg file)

**Comment by vvzpaco** [comment]
This has been happening since the last updates on the RG35XX PLUS, I thought it would have been fixed

**Comment by acmeplus** [comment]
No, it has not been fixed, all the controls changed a couple of versions ago when the firmware was consolidated for both RG35XX Plus and H.

**Comment by vvzpaco** [comment]
Is there any way I can revert only the controls in the new updates?
